core/shared/src/main/scala/scalaz/zio/stream/package.scala

```diff
@@ -0,0 +1,19 @@
+package scalaz.zio
+
+package object stream {
+
+  type Stream[+E, +A] = ZStream[Any, E, A]
+
+  type StreamPure[+A] = StreamPureR[Any, A]
+  val StreamPure = StreamPureR
+
+  type Sink[+E, +A0, -A, +B] = ZSink[Any, E, A0, A, B]
```

Comment:
@iravid What do you think about making further simplifications here? e.g. `A0 =:= A`?

Reply:
Yes, let's do that!